lang/en.json

```diff
@@ -778,8 +778,9 @@
         "message": "Store is currently unavailable, please try again in a few minutes"
     },
     "forbidden": {
-        "page_heading": "403 Error - Forbidden",
-        "message": "You do not have permission to access this page."
+        "page_heading": "Sorry! Please log in to continue",
+        "message": "You do not have permission to access this page. If you were logged in, ",
+        "sign_in": "please sign in"
```

Comment:
`If you were logged in, please sign in` does not make sense to me. Is this the actual copy or this is going to change ?